META6.json

```diff
@@ -8,7 +8,8 @@
     "JSON::Pretty",
     "Pod::To::Markdown:ver<0.1.4+>",
     "Shell::Command",
-    "CPAN::Uploader::Tiny:ver<0.0.4+>"
+    "CPAN::Uploader::Tiny:ver<0.0.4+>",
+    "TAP::Harness"
```

Comment:
It seems that travis ci issue will be fixed if we replace this by `TAP:ver<0.0.8+>`.

Reply:
That is unexpected, but if it works it works